catalog/ui/catalog-ui-search/src/main/webapp/component/dropdown/search-type/dropdown.search-type.hbs

```diff
@@ -14,5 +14,5 @@
 <div class="interaction-icon fa fa-file">
 </div>
 <div class="interaction-text">
-    Change Type
+    Use another Search Form
```

Comment:
nit: Should A in "another" be capitalized since everything is title case?

Reply:
thanks @bdeining!  I am +1 now.

Reply:
fixed capitalization